performance-test/run_performance_test.sh

```diff
@@ -8,7 +8,8 @@ APICONFIG_SUBSCRIPTION_KEY=$5
 GPD_SUBSCRIPTION_KEY=$6
 GPS_SUBSCRIPTION_KEY=$7
 DONATIONS_SUBSCRIPTION_KEY=$8
-DEBT_POSITION_NUMBER=$9
+SOAP_SUBSCRIPTION_KEY=$9
```

Comment:
Maybe you could consider using a single variable for all micro-services as in the integration tests